src/python/turicreate/toolkits/activity_classifier/_tf_model_architecture.py

```diff
@@ -21,13 +21,19 @@
 
 class ActivityTensorFlowModel(TensorFlowModel):
 
-    def __init__(self, net_params, batch_size, num_features, num_classes, prediction_window, seq_len):
+    def __init__(self, net_params, batch_size, num_features, num_classes, prediction_window, seq_len, config):
 
+        _utils.suppress_tensorflow_warnings()
         self.gpu_policy = _utils.TensorFlowGPUPolicy()
         self.gpu_policy.start()
 
         for key in net_params.keys():
             net_params[key] = _utils.convert_shared_float_array_to_numpy(net_params[key])
+        if 'random_seed' in config:
+            seed = int(_utils.convert_shared_float_array_to_numpy(config['random_seed']))
+            _tf.set_random_seed(seed)
```

Comment:
What is this seed used for? Aren't we already specifying a seed for the dropout layer?

Reply:
`_tf.set_random_seed(seed)` sets the seed at graph level and setting the seed for dropout level is more op-level. And I have realized that it is redundant to call it here and it is sufficient to specify only at dropout layer.